app/code/Magento/CatalogImportExport/Test/Unit/Model/Import/Product/ValidatorTest.php

```diff
@@ -108,7 +112,7 @@ public function attributeValidationProvider()
             [
                 'any_behavior',
                 [],
-                ['product_type' => 'any'],
+                ['product_type' => 'any', 'sku' => ''],
```

Comment:
Are there any failures without specified `sku` ? Could this case be intentional ?

Reply:
Hello @xmav, thanks for your review, I checked this case, and think this change not need, because `Magento\Framework\Stdlib\StringUtils` was Mock, and always returned **`NULL`**. After this [change](https://github.com/magento/magento2/pull/34704/files/6c2d9374ca73aed00af567c1c76239ca42c2198b#diff-268adde0c7b547267bf43dee0817df1ff69ffd413ebd339b1c1da6cb5a6e4657R69)  method `Magento\Framework\Stdlib\StringUtils::cleanString($string)` does not return **`NULL`**.

Thank you